project-base/.env.test

```diff
@@ -1 +1,14 @@
 # define your env variables for the test environment here
+
+DATABASE_HOST=postgres
+DATABASE_PORT=5432
+DATABASE_NAME=shopsys-test
+DATABASE_USER=root
+DATABASE_PASSWORD=root
```

Comment:
You do not need to have duplicated variables here. Even in `TEST` environment, `.env` file is loaded first, so you do not need to override variables that are the same

Reply:
I am confused how should the .env file should look like. In this comment you mind redundant ENVs but in comment bellow you want to introduce more reduntant ENVs. 🤔

Reply:
Oh, I see. I want to remove redundant parameters, but introduce prefixes that should be different by default (probably with `test-` prefix).

Reply:
I am for removing the reduntant ENVs. It means also not introducing test prefixes as we do not have them configured. Maybe we should get it done in following PR. Are you OK with that?

Reply:
Redundats were removed in 2f196da2d7881ea636a8c0972216fb5f366dea27